src/Microsoft.ML.Data/Evaluators/EvaluatorUtils.cs

```diff
@@ -1553,13 +1581,13 @@ private static string GetFoldMetricsAsString(IHostEnvironment env, IDataView dat
         }
 
         // Get a string representation of a confusion table.
-        private static string GetConfusionTableAsString(double[][] confusionTable, double[] rowSums, double[] columnSums,
-            List<ReadOnlyMemory<char>> predictedLabelNames, string prefix = "", bool sampled = false, bool binary = true)
+        internal static string GetConfusionTableAsString(ConfusionMatrix confusionMatrix, bool isWeighted)
```

Comment:
>GetConfusionTableAsString [](start = 31, length = 25)

Mix of `ConfusionMatrix` and `ConfusionTable` makes me 
...
[http://i.imgur.com/lgSDkvC.gif](http://i.imgur.com/lgSDkvC.gif)
...
Confuse!

Reply:
there is also confusionDataView

---
In reply to: [275081264](https://github.com/dotnet/machinelearning/pull/3250#discussion_r275081264) [](ancestors = 275081264)